api-explorer/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "_from": "swagger-ui-dist@^3.4.4",
+  "_id": "swagger-ui-dist@3.4.4",
+  "_inBundle": false,
+  "_integrity": "sha1-fdDX94hOQ4e50Q1PT3sPk0NqEdI=",
+  "_location": "/swagger-ui-dist",
+  "_phantomChildren": {},
+  "_requested": {
+    "type": "range",
+    "registry": true,
+    "raw": "swagger-ui-dist@^3.4.4",
+    "name": "swagger-ui-dist",
+    "escapedName": "swagger-ui-dist",
+    "rawSpec": "^3.4.4",
+    "saveSpec": null,
+    "fetchSpec": "^3.4.4"
+  },
+  "_requiredBy": [
+    "#DEV:/"
+  ],
+  "_resolved": "https://registry.npmjs.org/swagger-ui-dist/-/swagger-ui-dist-3.4.4.tgz",
+  "_shasum": "7dd0d7f7884e4387b9d10d4f4f7b0f93436a11d2",
+  "_spec": "swagger-ui-dist@^3.4.4",
+  "_where": "/Users/rfeng/Projects/docs/loopback.io",
```

Comment:
Hmm, this refers to your local checkout, I find that suspicious.

This whole `package.json` file looks to me like it's coming from `node_modules/swagger-ui` after running `npm install swagger-ui`. Is that intentional?

Reply:
Can we modify `update-swagger-ui.sh` to skip copying `package.json`?